src/components/pages/AppOverview/ChangeDocuments.tsx

```diff
@@ -0,0 +1,22 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+export default function ChangeDocuments() {
+  return <>In Progress</>
```

Comment:
Can you move this to shared folder and make it more reusable? better name for the file

Reply:
This feature is new and I don't think this needs to be in shared components

Reply:
concern is not to hard code any text and give a common name to the file. We might see another text in place of `IN PROGRESS` in future. Current file name and the content is totally different